src/createMixin.ts

```diff
@@ -27,33 +16,10 @@ export default (options: Options = {}): JsonldMixin => {
   };
 
   return {
-    head(this: Vue) {
-      if (!this.$options || typeof this.$options.jsonld !== 'function') {
-        return {};
-      }
-
-      if (this.$options.jsonld.call(this) === null) {
-        return {};
+    beforeCreate (this: Vue) {
+      if (this.$options && typeof this.$options.jsonld === 'function') {
+        this.$options.head = mergeHead.call(this, mergedOptions)
```

Comment:
I am slightly worried that you will loose the reactivity of head with this approach, did you check that?

It would probably be an improvement to set the original head function as a second computed prop here. By returning a function in mergeHead vue-meta will always create a computed prop `$metaInfo`. Use that same approach here for the original head (if its a function) and create another computed prop in your mergeHead function. That way you never have to call a head function yourself (you can use the computed prop directly) and because you use the computed prop reactivity should not be lost.

Reply:
Another benefit of using a computed prop is that Vue takes care of caching. If you call the head fn manually, the fn will be called everytime the meta of any component is updated. A computed prop is cached in Vue and only updates when one its own 'dependencies' changes.

Reply:
Done in the new commit! Set the computed property for head and also for LD-JSON, even if i don't see any meaningful usage for reactive json-ld 🤔